src/lib/utils.js

```diff
@@ -0,0 +1,10 @@
+export function sleep(ms) {
```

Comment:
use `import { setTimeout as sleep } from "timers/promises"; }` instead

Reply:
Didn't know that nodejs API.
I like it, just removed that sleep from utils and imported the `setTimeout` directly on `ec2Handler.js`